### PR TITLE
neil: init at 0.0.13

### PR DESCRIPTION
**Pull request by jlesquembre**
###### Motivation for this change
Add [neil](https://github.com/babashka/neil) to nixpkgs

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/tools/neil/default.nix [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  src = fetchFromGitHub {
+    owner = "babashka";
+    repo = "neil";
+    rev = "v${version}";
+    sha256 = "0jiyl0d39d8kk5bpangwxiy90vqipj4lgp8x84rh4z5m53knjpkd";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  buildPhase = null;
```
Comment (figsoda) [inline pkgs/development/tools/neil/default.nix:21]:
```suggestion
  dontBuild = true;
```

File: pkgs/development/tools/neil/default.nix [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp neil $out/bin
```
Comment (figsoda) [inline pkgs/development/tools/neil/default.nix:25]: maybe `install` could be better?

File: pkgs/development/tools/neil/default.nix [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    wrapProgram $out/bin/neil \
+      --prefix PATH : "${lib.makeBinPath [ babashka ]}"
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/babashka/neil";
+    description = "A CLI to add common aliases and features to deps.edn-based projects";
+    license = licenses.epl10;
+    platforms = babashka.meta.platforms;
+    maintainers = [ maintainers.jlesquembre ];
```
Comment (figsoda) [inline pkgs/development/tools/neil/default.nix:35]:
```suggestion
    maintainers = with maintainers; [ jlesquembre ];
```

File: pkgs/development/tools/neil/default.nix [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    mkdir -p $out/bin
+    cp neil $out/bin
+    wrapProgram $out/bin/neil \
+      --prefix PATH : "${lib.makeBinPath [ babashka ]}"
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/babashka/neil";
+    description = "A CLI to add common aliases and features to deps.edn-based projects";
+    license = licenses.epl10;
```
Comment (figsoda) [inline pkgs/development/tools/neil/default.nix:33]: upstream doesn't seem to have a license yet
https://github.com/babashka/neil#license

**Review by figsoda — CHANGES_REQUESTED** [review]

File: pkgs/development/tools/neil/default.nix (hunk 2 above)
Comment (jlesquembre) [reply pkgs/development/tools/neil/default.nix]: true, updated

File: pkgs/development/tools/neil/default.nix (hunk 4 above)
Comment (jlesquembre) [reply pkgs/development/tools/neil/default.nix]: right, now there is a license (MIT)

**Review by figsoda — APPROVED** [review]

**Comment by figsoda** [comment]
Result of `nixpkgs-review pr 139883` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>neil</li>
  </ul>
</details>

**Pull request merged**